.circleci/config.yml

```diff
@@ -3,8 +3,49 @@ notify-on-fail: &notify-on-fail
     condition: always
     steps:
       - slack/notify:
-          event: fail
-          template: basic_fail_1
+          event: always
```

Comment:
This was added to test the notification, we need to remove it once we know it works, and before this PR is merged

Reply:
restored in https://github.com/sparkletown/sparkle/pull/786/commits/02cd75177a998f4182269cca99a79d7f55ce85fe